src/annotator/plugin/test/bucket-bar-test.js

```diff
@@ -0,0 +1,227 @@
+import $ from 'jquery';
```

Comment:
Except for mocking and that kind of thing, these are generally unchanged, just decaffeinated.